modules/olm-disabling-copied-csvs.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-config.adoc
+
+:_content-type: PROCEDURE
+[id="olm-disabling-copied-csvs_{context}"]
+= Disabling copied CSVs
+
+When an Operator is installed by Operator Lifecycle Manager (OLM), a simplified copy of its cluster service version (CSV) is created in every namespace that the Operator is configured to watch. These CSVs are known as _copied CSVs_ and communicate to users which controllers are actively reconciling resource events in a given namespace.
+
+When Operators are installed with the `AllNamespaces` install mode, a copied CSV is created in every namespace on the cluster. On especially large clusters, with namespaces and installed Operators potentially in the hundreds or thousands, copied CSVs consume an untenable amount of resources, such as OLM's memory usage, cluster etcd limits, and networking.
```

Comment:
Are these two different ways an Operator can be installed? Or is the second paragraph a component of the first paragraph?

Either way, I'd either add a little context.

Now, looking at the next sentence (Line 13), I wonder if the first paragraph is needed. Especially if these two paragraphs are different things.

Reply:
2nd para is a component of the 1st. I'll clarify.